### DISCUSSION TITLE
python color different from vs code

### DISCUSSION THREAD

**Discussion by marslogin** [coder/code-server#4565] [Q&A] [answered]
could you please help?
code server: v3.12.0, standalone run on ubuntu 20.04
![image](https://user-images.githubusercontent.com/45973915/143944295-945edf60-3de0-4903-a480-5ee2e27d93ef.png)
vs code: v1.62.3
![image](https://user-images.githubusercontent.com/45973915/143944315-a30d7c35-6229-4f19-b671-a3728e3df1c3.png)

**Comment by lg-nineoneninethreenine** [comment]
Oh, the code colouring, eh? Can you try checking for anything related to code colouring in your settings(<kbd>F1</kbd> > Settings(UI) > Search for code colouring)?

**Reply by marslogin** [reply]
I am sorry to say, after I check, I don't find such things. Please don't mind i'm a novice

**Reply by lg-nineoneninethreenine** [reply]
Hmm... Can't get my instance on, that's too slow, and I use Windows 7... Hmm... I will check on this and come back.

**Comment by shp7724** [comment, marked as answer]
code-server does not officially support PyLance, and that's why you aren't getting the same code highlighting (and intellisense) as in VSCode.

**Reply by marslogin** [reply]
ok, thank you for your answer
